### PR TITLE
Assignment9_First_Commit

### PR DESCRIPTION
**Pull request by GobindaFexleTrainee**
Added objects,tabs,customfields and layouts

### CODE REVIEW

File: src/package.xml [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
+        <members>Subject__c</members> 
+        <members>Teacher__c</members> 
+        <members>Year_of_Class__c</members>
+        <members>College__c</members>
         <name>CustomTab</name>
     </types>
     <types>
-        <members>Student__c-Registrar Layout</members>
-        <members>Student__c-Teacher Layout</members>
+        <members>Pan_Card__c-Pan Card Layout</members>
```
Comment (shubhamsharmafexletraining) [inline src/package.xml]: Why were these layouts created? these are not requested in the assignment for now. Remove them all.

File: src/package.xml [hunk 2]
```diff
@@ -1,21 +1,33 @@
...
+        <members>Student__c</members>
+        <members>StudentSubjects__c</members>
+        <members>Subject__c</members> 
+        <members>Teacher__c</members> 
+        <members>Year_of_Class__c</members>
+        <members>College__c</members>
         <name>CustomObject</name>
     </types>
     <types>
         <members>Student__c</members>
```
Comment (shubhamsharmafexletraining) [inline src/package.xml]: Student Tab is already deployed. remove it from the Package.xml

File: src/package.xml [hunk 3]
```diff
@@ -1,21 +1,33 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Package xmlns="http://soap.sforce.com/2006/04/metadata">
-    <types>
-        <members>Insert_Student_Details</members>
-        <members>StudentQuery</members>
-        <name>ApexClass</name>
-    </types>
     <types>
         <members>Student__c</members>
```
Comment (shubhamsharmafexletraining) [inline src/package.xml]: Student Object is already deployed. remove it from the Package.xml

File: src/package.xml [hunk 4]
```diff
@@ -1,21 +1,33 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Package xmlns="http://soap.sforce.com/2006/04/metadata">
-    <types>
-        <members>Insert_Student_Details</members>
-        <members>StudentQuery</members>
-        <name>ApexClass</name>
-    </types>
```
Comment (shubhamsharmafexletraining) [inline src/package.xml:7]: Add custom fields for the relationship for Objects which are already there/Deployed in Production Org but new relationship fields are created for them like **Contact** & **Pan Card** and **Lead** & **Account**.

File: src/package.xml [hunk 5]
```diff
@@ -1,21 +1,33 @@
...
+        <members>Year_of_Class__c</members>
+        <members>College__c</members>
         <name>CustomObject</name>
     </types>
     <types>
         <members>Student__c</members>
+        <members>Pan_Card__c</members>
+        <members>Student__c</members>
+        <members>StudentSubjects__c</members>
+        <members>Subject__c</members>
```
Comment (shubhamsharmafexletraining) [inline src/package.xml:16]: Subject__c.tab is not added in this PR. Include the tab component also.

**Review by shubhamsharmafexletraining — CHANGES_REQUESTED** [review]
Fix the Pr comments

File: src/package.xml (hunk 3 above)
Comment (GobindaFexleTrainee) [reply src/package.xml]: student object modified For assignment 9

Comment (shubhamsharmafexletraining) [reply src/package.xml]: If Student Object is modified for Assignment 9 then please add the changes for the student object in this PR.

**Pull request closed without merging**